components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml

```
@@ -52,3 +52,4 @@ spec:
         - integration-service
         - ci-helper-app
         - mintmaker
+        - tracing
```

the namespace is `konflux-otel `, not `tracing`.
```suggestion
        - konflux-otel 
```

---

TYVM, fixing